tests/flows/test_inference.py

```diff
@@ -458,23 +458,26 @@ def test_remove_sabin_file_stems(
 
 
 def test_group_inference_results_into_states(snapshot):
```

Comment:
```suggestion
def test_get_inference_results_as_state_groups(snapshot):
```

I feel that the name of the method and subsequent test method could be made clearer